notebooks/DemoNotebook_interactive_plots.ipynb

```diff
@@ -71,16 +73,29 @@
    "outputs": [],
    "source": [
     "# analyse small dataset\n",
-    "data_manager = DataManager(\"../data/Test_Data/XMI_11\")\n",
+    "data_manager = DataManager(\"/content/data/Test_Data/XMI_11\")\n",
     "# analyse full dataset\n",
     "# data_manager = DataManager(\"/content/data/All_Data/XMI_11\") "
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The integrity of the data is checked using the `check_data_integrity` method:"
+    "The integrity of the data is checked using the `check_data_integrity` method:\n",
```

Comment:
From the user's side, it would be more logical if `check_data_integrity` would return `True` if the data quality is high, and `False` if the quality is poor. So the function should return `data_integrity = True` for good data and `data_integrity = False` for bad data.
```
                if under_threshold_dict:
                    logging.warning(warning_str)
                    data_integrity = False

        return data_integrity
```